### PR TITLE
Update dotnet supported runtimes for AWS Lambda

### PR DESCRIPTION
**Pull request by vasireddy99**
This PR is created to update the supported runtimes for AWS Lambda.[ Supported runtimes in AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html)

Note: This [PR](https://github.com/open-telemetry/opentelemetry-lambda/pull/434) in OTel-lambda needs to be merged

### CODE REVIEW

**Comment by bryan-aguilar** [comment]
Is this blocked by the linked PR and also a new lambda layer release that includes the change?

**Comment by vasireddy99** [comment]
> Is this blocked by the linked PR and also a new lambda layer release that includes the change?

Yes this blocked by the linked PR. but wouldn't necessarily require a new release as the layer is not modified for this purpose other than the sample app and integ-tests.

**Comment by vasireddy99** [comment]
This PR is Ready for Review. below is the following result to give us enough confidence for stating in the website.

Trace in X-Ray: 
image

Integration test local : 
image

**Review by PaurushGarg — APPROVED** [review]

File: src/docs/getting-started/lambda/lambda-dotnet.mdx [hunk 1]
```diff
@@ -21,7 +21,7 @@ This chapter will walk you through the steps to manually instrument your Lambda
 
 ## Requirements
 
-The ADOT Lambda .NET SDK supports `dotnetcore2.1` and `dotnetcore3.1` Lambda runtimes.
+The ADOT Lambda Layer for .Net supports both `dotnetcore3.1` and `dotnet6` Lambda runtimes.
```
Comment (Aneurysm9) [inline src/docs/getting-started/lambda/lambda-dotnet.mdx:24]: Is this correct?  We don't have an "ADOT Lambda Layer for .NET", but a collector-only layer that can be used with .NET.  What about this:

```suggestion
The OpenTelemetry Lambda SDK  for .NET supports both `dotnetcore3.1` and `dotnet6` Lambda runtimes.
```

Maybe also adjust other references to excise `ADOT` or `AWS Distro for`.

Comment (vasireddy99) [reply src/docs/getting-started/lambda/lambda-dotnet.mdx]: Thank you, used the suggestions

**Pull request merged**